java/code/src/com/redhat/rhn/frontend/servlets/test/I18NFilterTest.java

```diff
@@ -42,7 +50,7 @@ public void testCheckCharset() throws Exception {
         }
         catch (IOException ioe) {
             // This should never happen ..
-            throw new Exception("doFilter() failed ..");
+            fail("doFilter() failed ..");
```

Comment:
I think we can also just remove the catch here, the test will fail anyway and the message is not more informative than a stacktrace :wink: